docs/upgrading.md

```diff
@@ -5,6 +5,13 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open Autonomy
 
+## `v0.5.0.post2` to `v0.6.0`
+
+Breaking changes
+
+- The pre and post conditions needs to be defined while the FSM
```

Comment:
```suggestion
- Optional: The developer can update to use the auto-generated behaviour and round ids. In this case, the `behaviour_id` and `round_id` need no longer be set on the class. When accessing the auto generated ids on the class use `auto_behaviour_id()` and `auto_round_id()` class methods. On the instance `behaviour_id` and `round_id` properties resolve the auto id class methods, so no change is required.
- Required: The pre- (`db_pre_conditions`) and post- (`db_post_conditions`) conditions on the synchronized data need to be defined for each initial and final state in an `AbciApp`. See `transaction_settlement_abci` for an example.
```

Reply:
@Adamantios suggest you add a couple of lines in the docs on the pre and post conditions and the default conditions

Reply:
Done in ae17700d472b8c5317f959327a9ebf4de1aed314.